pkg/usageStats/stats.go

```diff
@@ -583,6 +602,32 @@ func parseStatsRecord(record []string) (ReadStats, error) {
 			return readStats, fmt.Errorf("parseStatsRecord: could not parse MetricsBytesCount field '%v': %w", record[5], err)
 		}
 		readStats.TimeStamp = time.Unix(tsString, 0)
+	case 8:
+		readStats.MetricsDatapointsCount, err = strconv.ParseUint(record[2], 10, 64)
+		if err != nil {
+			return readStats, fmt.Errorf("parseStatsRecord: could not parse MetricsDatapointsCount field '%v': %w", record[2], err)
+		}
+		tsString, err := strconv.ParseInt(record[3], 10, 64)
+		if err != nil {
+			return readStats, fmt.Errorf("parseStatsRecord: could not parse timestamp field '%v': %w", record[3], err)
+		}
```

Comment:
This whole switch statement is getting messy and repetitive. We might want to find a way to make this cleaner.

However, I think for this PR you can keep what you have